src/function.js

```diff
@@ -1,10 +1,35 @@
 /** @license MIT License (c) copyright 2010-2016 original author or authors */
 
 // id :: a -> a
-export const id = x => x;
+export const id = x => x
 
 // compose :: (b -> c) -> (a -> b) -> (a -> c)
-export const compose = (f, g) => x => f(g(x));
+export const compose = (f, g) => x => f(g(x))
 
 // apply :: (a -> b) -> a -> b
-export const apply = (f, x) => f(x);
+export const apply = (f, x) => f(x)
+
+// curry2 :: (a -> b -> c) -> a -> b -> c
```

Comment:
I don't think there's a right answer :)  Pick whichever you feels like conveys the intent the best, and let's go with it.